src/archivematica/storage_service/storage_service/settings/base.py

```diff
@@ -659,7 +666,13 @@ def _get_email(email):
     OIDC_USERNAME_ALGO = _get_email
 
     # map attributes from access token
-    OIDC_ACCESS_ATTRIBUTE_MAP = {"given_name": "first_name", "family_name": "last_name"}
+    default_claims = {"given_name": "first_name", "family_name": "last_name"}
+    try:
+        OIDC_ACCESS_ATTRIBUTE_MAP = json.loads(
+            environ.get("OIDC_ACCESS_ATTRIBUTE_MAP", default_claims)
+        )
+    except json.JSONDecodeError:
+        OIDC_ACCESS_ATTRIBUTE_MAP = default_claims
```

Comment:
Should the README state that the value of the map needs to be JSON-encodable?

Reply:
I have added this note

Reply:
Sorry for being misleading here @sbreker, but I think the note should read `JSON-decodable`?